test/deploy.js

```diff
@@ -91,3 +92,23 @@ test.serial('deployAll --prune', (t) => {
     }))
     .then(() => deploy.deployAll({ prune: true }));
 });
+
+test.serial('deployAll with .DS_Store files', (t) => {
```

Comment:
There doesn't seem to be any assertions in this test, can we trust that no rejection on the returned promise means that it was successful?